packages/csharp/PortingAssistant/PortingAssistant.Api/Application.cs

```diff
@@ -111,18 +118,54 @@ public void SetupConnection(bool console = false)
                 try
                 {
                     var file1 = httpService.DownloadS3FileAsync("newtonsoft.json.json.gz");
-                    file1.Wait();
                     var file2 = httpService.DownloadS3FileAsync("52projects.json.gz");
-                    file2.Wait();
                     var file3 = httpService.DownloadS3FileAsync("2a486f72.mega.json.gz");
-                    file3.Wait();
-                    return true;
+                    Task.WhenAll(file1, file2, file3).Wait();
+                    return file1.IsCompletedSuccessfully || file2.IsCompletedSuccessfully || file3.IsCompletedSuccessfully;
                 }
                 catch
                 {
                     return false;
                 }
             });
+
+            _connection.On<CustomerFeedbackRequest, Response<bool, string>>("sendCustomerFeedback", request =>
+            {
+                try
+                {
+                    string endPoint = _cccongif.CustomerFeedbackEndpoint;
+                    string uniqueMachineID = LogUploadUtils.getUniqueIdentifier();
+                    string key = $"{uniqueMachineID}/{request.Date}";
```

Comment:
is there any specific format we need to use for date to be consistent all the time. Refer this https://www.c-sharpcorner.com/blogs/date-and-time-format-in-c-sharp-programming1

Reply:
I amended codes in front-end side (packages/react/src/components/CustomerContribution/CustomerFeedbackModal.tsx, line 64) to use ISO format date (such as 2021-10-05T14:48:00.000Z).